projects/mysadna/src/app/components/user-details/user-details.component.html

```diff
@@ -27,22 +27,22 @@ <h1 class="user-header">{{user.firstName}} {{user.lastName}}</h1>
         <h3 class="projects-header">Projects:</h3>
         <ul class="projects-list">
           <li *ngFor="let project of userProjects">
-            <a [routerLink]="'/project-details/' + project.projectId" class="project">{{project.projectName}}</a>
+            <a [routerLink]="'/project-details/' + project.getProjectId()" class="project">{{project.getProjectName()}}</a>
```

Comment:
Line is too big.
Please try to not reach 100 chars per line

Reply:
Right.